client/app/components/onboardingGuide/GuideStatusPage.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideStatusPage = (props) => {
+
+  const handleClick = function(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  }
+
+  const onboarding_data = props.onboarding_data;
+  // TODO: interpolating translation strings needs more thinking
+  const title = props.nextStep ? props.t('title').replace(/\%\{(\w+)\}/g, props.name) : props.t('title_done');
+
+
+  const todo_desc_partial =
+    <div>
+      <p className={ scss.description } >
+        { props.t('description_p1') }
+      </p>
+      <p className={ scss.description } >
+        { props.t('description_p2') }
+      </p>
+    </div>
+
+  const done_desc_partial =
+    <p className={ scss.description }
+      dangerouslySetInnerHTML={{ __html: props.t('congratulation') }} ></p>
```

Comment:
Why is `dangerouslySetInnerHTML` used here? If it avoids the extra `<span>` elements within the `<p>` tag, rebasing to the `react-on-rails` branch updates React to 15.0.1 which doesn't generate the extra spans anymore.